diamm/serializers/website/city.py

```diff
@@ -29,12 +33,31 @@ class Meta:
 class CityDetailSerializer(serializers.HyperlinkedModelSerializer):
     archives = ArchiveCitySerializer(many=True)
     country = CountryCitySerializer(
-        source="parent"
+        source='parent'
     )
+    sources = serializers.SerializerMethodField() 
 
     class Meta:
         model = GeographicArea
-        fields = ('url', 'name', 'archives', 'country')
+        fields = ('url', 'name', 'archives', 'country', 'sources')
         extra_kwargs = {
             'url': {'view_name': 'city-detail'}
         }
+
+    def get_sources(self, obj):
+        sources = list()
+        connection = pysolr.Solr(settings.SOLR['SERVER'])
+        fq = ['type:source', 'geographic_area_ii:' + str(obj.id)]
```

Comment:
The general pattern is `geographic_area_ii:{0}'.format(obj.id)` which will cast it to a `str` in the formatting process.